python_modules/dagster/dagster/_check/__init__.py

```diff
@@ -867,6 +867,10 @@ def is_list(
 # ########################
 
 
+def is_mapping_param(obj: object) -> bool:
```

Comment:
I feel like this doesn't quite align with other functions of the check library, which would usually raise an error if the thing was not of the expected type. seems ok to just inline this functionality if it's only being used once